onnxruntime/core/providers/tensorrt/tensorrt_execution_provider.cc

```diff
@@ -3752,6 +3752,11 @@ Status TensorrtExecutionProvider::CreateNodeComputeInfoFromGraph(const GraphView
       trt_context = trt_state->context->get();
     }
 
+    // Check before using trt_engine
+    if (trt_engine == nullptr) {
+      return ORT_MAKE_STATUS(ONNXRUNTIME, EP_FAIL, "Invalid TensorRT engine, please rebuild.");
```

Comment:
The error message seems not quite accurate.
trt_engine is null pointer means engine eight not being successfully built or not successfully loaded from engine cache.
I think we can simply output "No engine is found"

Reply:
Please also add this null pointer check inside the compute_info.compute_func of CreateNodeComputeInfoFromPrecompiledEngine()

Reply:
Comments applied